stock_picking_show_returns/__openerp__.py

```diff
@@ -1,44 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
Isn't it better to keep old module name? Although it has an `s` at the end, you afford some incompatibility issues